### ISSUE TITLE
No Code

### ISSUE THREAD

**Issue by khqi-thu** [alexandrosgk/sparse_coding_deep_equilibrium#2]
Dear,
I have read your paper, but I cannot find the code here.
Is there any way to get the code?

**Comment by alexandrosgk** [comment]
Yes, in the following month we are planning to release our source code. The release has been delayed partly due to a heavy workload and numerous deadlines in our current projects.

**Comment by khqi-thu** [comment]
> Yes, in the following month we are planning to release our source code. The release has been delayed partly due to a heavy workload and numerous deadlines in our current projects.

Thanks. Looking forward to your code!

**Comment by junhanchen912** [comment]
Dear Dr. Alexandros
Recently, I read your article "Connections Between Deep Equilibrium and Sparse Representation Models With Application to Hyperspectral Image Denoising", and I was very interested in the deep equilibrium sparse coding method you mentioned, which greatly improved the interpretibility of deep neural networks in the field of image denoising, and reduced the computational complexity of the depth expansion of sparse coding by using the deep equilibrium model. I benefited a lot from your article.
In the article, I saw that you disclosed your deep equilibrium sparse coding model on github, but when I read your article, you have deleted the model code, I don't know what the reason is, but from my personal point of view, your article has inspired me a lot, I wonder if you can share the model code for me to further learn the model.
I'm sorry for being so presumptuous.I wish you a smooth scientific research and a happy life.

**Comment by ChentongWang** [comment]
Dear Dr.Gkillas,
       I am writing to inquire if you would be willing to share the source code of your work for my reference. Access to your code would greatly assist me in furthering my understanding of the study. Thank you for your consideration. I wish you continued success in your research endeavors.
